### PR TITLE
Bug 1884643 update resolving multi-attach errors procedure

### PR DESCRIPTION
**Pull request by bmcelvee**
https://bugzilla.redhat.com/show_bug.cgi?id=1884643

@vikram-redhat @bobfuru fyi

### CODE REVIEW

File: modules/storage-multi-attach-error.adoc [hunk 1]
```diff
@@ -28,4 +30,11 @@ For most storage solutions, you can use ReadWriteMany (RWX) volumes to prevent m
 +
 For storage that does not support RWX, such as VMware vSphere, RWO volumes must be used instead. However, RWO volumes cannot be mounted on multiple nodes.
 +
-If you encounter a multi-attach error message with an RWO volume, you can either recover or delete the failed node to make the volume available to other nodes.
+If you encounter a multi-attach error message with an RWO volume, you can force delete the pod on a shutdown or crashed node.
```
Comment (kalexand-rh) [inline modules/storage-multi-attach-error.adoc:33]: If this is the way to fix it, I'd ~you can~.
```suggestion
If you encounter a multi-attach error message with an RWO volume, force delete the pod on a shut down or crashed node.
```

File: modules/storage-multi-attach-error.adoc [hunk 2]
```diff
@@ -28,4 +30,11 @@ For most storage solutions, you can use ReadWriteMany (RWX) volumes to prevent m
...
++
+[source,terminal]
+----
+$ oc delete pod <old_pod> --force=true --grace-period=0s
+----
++
+This deletes the volumes stuck on shutdown or crashed nodes after six minutes.
```
Comment (kalexand-rh) [inline modules/storage-multi-attach-error.adoc:40]:
```suggestion
This command deletes the volumes stuck on shutdown or crashed nodes after six minutes.
```

**Review by kalexand-rh — COMMENTED** [review]
Two small suggestions, but it LGTM

**Comment by bmcelvee** [comment]
No QE review required.

**Comment by bobfuru** [comment]
Thanks for fixing, @bmcelvee. I also created and merged PR 27719 to update an instance of this wording in the persistent storage docs.

**Pull request merged**